scripts/pre-push.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Run as many lints as we can without taking up too much time.
+# NOTE: we use npx and node instead of pnpm because it's about twice as fast to startup.
```

Comment:
To make it _even_ faster you can execute `node_modules/.bin/dprint` and co directly to avoid the npx overhead :)